scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -86,6 +86,26 @@ workflow GvsCreateVATfromVDS {
     String effective_vat_version = if (defined(vat_version) && select_first([vat_version]) != "v1") then "_" + select_first([vat_version]) else ""
     String vat_table_name = filter_set_name + "_vat" + effective_vat_version
 
+    call Utils.GetBQTableLastModifiedDatetime as SampleDateTime {
```

Comment:
I would suggest that if `split_intervals_scatter_count` is defined, just use it and don't look up the number of samples in BQ. It's possible that wdl/cromwell is smart enough to do that?

Reply:
Do you have an example that doesn't involve coercing an optional `Int` to a non-optional `Int`?

Reply:
I *think* this would work?

```suggestion
    if (!defined(split_intervals_scatter_count)) {
        call Utils.GetBQTableLastModifiedDatetime as SampleDateTime {
            input:
                project_id = project_id,
                fq_table = "~{project_id}.~{dataset_name}.sample_info",
                cloud_sdk_docker = effective_cloud_sdk_docker,
        }

        call Utils.GetNumSamplesLoaded {
            input:
                fq_sample_table ="~{project_id}.~{dataset_name}.sample_info",
                project_id = project_id,
                sample_table_timestamp = SampleDateTime.last_modified_timestamp,
                cloud_sdk_docker = effective_cloud_sdk_docker,
        }

        Int calculated_scatter_count = if (GetNumSamplesLoaded.num_samples < 11) then 10 else
                                       if(GetNumSamplesLoaded.num_samples < 250000) then 500 else
                                       if(GetNumSamplesLoaded.num_samples < 450000) then 1000 else 2000
    }

    Int effective_scatter_count = select_first([split_intervals_scatter_count, calculated_scatter_count])```

Reply:
cool - thanks!